exporter/clickhouseexporter/internal/exponential_histogram_metrics.go

```diff
@@ -51,6 +51,7 @@ CREATE TABLE IF NOT EXISTS %s_exponential_histogram %s (
     Flags UInt32  CODEC(ZSTD(1)),
     Min Float64 CODEC(ZSTD(1)),
     Max Float64 CODEC(ZSTD(1)),
+		AggTemp Int32 CODEC(ZSTD(1)),
```

Comment:
I think it's better use the same field name `AggregationTemporality` in opentelemetry specification.

Reply:
Also agree with this change. It's more to type, but it's good to keep close to the spec

Reply:
Updated `AggTemp` to `AggregationTemporality` for all metric types for which temporality is relevant.